tmasTest/templates/edit.html

```diff
@@ -2,27 +2,64 @@
 
 {% block content %}
 
-<!-- Create form for editing the location, date, subject line, and story -->
-
```

Comment:
I don't know why it says I deleted this whole section. It still in there, just indented.